pyproject.toml

```diff
@@ -41,8 +41,10 @@ pydantic = ">=2.8.2,<2.9.0"
 PyDispatcher = "^2.0.6"
 pytz = "^2022.6"
 tzlocal = "^4.2"
-mxcubecore = ">=1.191.0"
+mxcubecore =  "1.194.0"
```

Comment:
@marcus-oscarsson This might have been pinned by mistake, no? Can it be removed? `>=1.194.0` maybe?

Reply:
Yup, It it should indeed be `>=1.194.0` now

Reply:
Solved in <https://github.com/mxcube/mxcubeweb/pull/1520>.